lib/school.rb

```diff
@@ -0,0 +1,32 @@
+class School
+  attr_reader :start_time,
+              :hours,
+              :student_name
+
+  def initialize(start_time, hours)
+    @start_time = start_time
+    @hours = hours
+
+end
```

Comment:
Keep an eye on your indentation here - this `end` should line up directly under the `def`.  It might seem a bit trivial now, but good indentation can really help you read through your code and identify where things are going a little wonky.